runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -256,39 +644,42 @@ void TR::ARM64PrivateLinkage::createEpilogue(TR::Instruction *cursor)
    TR::RealRegister *javaSP = machine->getRealRegister(properties.getStackPointerRegister()); // x20
 
    // restore preserved GPRs
-   int32_t preservedRegisterOffset = cg()->getLargestOutgoingArgSize() + properties.getOffsetToFirstParm(); // outgoingArgsSize
-   TR::RealRegister::RegNum firstPreservedGPR = TR::RealRegister::x28;
-   TR::RealRegister::RegNum lastPreservedGPR = TR::RealRegister::x21;
-   for (TR::RealRegister::RegNum r = firstPreservedGPR; r >= lastPreservedGPR; r = (TR::RealRegister::RegNum)((uint32_t)r-1))
+   int32_t preservedRegisterOffsetFromJavaSP = cg()->getLargestOutgoingArgSize() + properties.getOffsetToFirstParm(); // outgoingArgsSize
+   TR::RealRegister::RegNum firstPreservedGPR = TR::RealRegister::x21;
+   TR::RealRegister::RegNum lastPreservedGPR = TR::RealRegister::x28;
+   for (TR::RealRegister::RegNum r = firstPreservedGPR; r <= lastPreservedGPR; r = (TR::RealRegister::RegNum)((uint32_t)r+1))
       {
       TR::RealRegister *rr = machine->getRealRegister(r);
       if (rr->getHasBeenAssignedInMethod())
          {
-         TR::MemoryReference *preservedRegMR = new (trHeapMemory()) TR::MemoryReference(javaSP, preservedRegisterOffset, cg());
+         TR::MemoryReference *preservedRegMR = new (trHeapMemory()) TR::MemoryReference(javaSP, preservedRegisterOffsetFromJavaSP, cg());
          cursor = generateTrg1MemInstruction(cg(), TR::InstOpCode::ldrimmx, lastNode, rr, preservedRegMR, cursor);
-         preservedRegisterOffset += 8;
+         preservedRegisterOffsetFromJavaSP += 8;
          }
       }
 
    // remove space for preserved registers
-   uint32_t frameSize = cg()->getFrameSizeInBytes();
-   if (constantIsUnsignedImm12(frameSize))
+   const TR::ARM64LinkageProperties& linkageProperties = getProperties();
```

Comment:
A variable `properties` is already declared and defined at the beginning of `createEpilogue()`.
I don't think you need another variable for `getProperties()`.